md/pages-development.md

```diff
@@ -0,0 +1,143 @@
+By downloading the Bonita User and Administrator applications from Bonita Studio Welcome page, you also download Bonita pages created with Bonita UI Designer.  
+Those pages can be customized and added to an application.
+
+## Custom pages
+A page resource has the general [resource definition](resource-management.md). 
+If it contains an `Index.groovy` file, this must implement the PageController interface optionally with libraries. 
+If you create a custom page with Bonita UI Designer, its required structure and content are right, by default.
+
+A custom page is displayed inside an iframe to prevent conflicts between Bonita Portal resources (for example JS and CSS) and those used in the custom page. 
+This also reduces the risk of migration issues, for example if a custom page uses the version of JQuery provided with Bonita Portal and it is updated.
+
+For all information about developping a custom page, go to the dedicated page
+
+### PageController interface
+```java
+public interface PageController {
+
+/**
+* Let the custom page parse request for specific attribute handling.
+*
+* @param request the HTTP servlet request intended to be used as in a servlet
+* @param response the HTTP servlet response intended to be used as in a servlet
+* @param pageResourceProvider provide access to the resources contained in the custom page zip
+* @param pageContext provide access to the data relative to the context in which the custom page is displayed
+*/
+public void doGet(HttpServletRequest request, HttpServletResponse response, PageResourceProvider pageResourceProvider, PageContext pageContext);
+}
+```
+
+### Permissions for custom pages
+
+If you are using [REST API authorization](rest-api-authorization.md) and your custom page is an HTML page using the Web REST API, 
+you need to specify the permissions that a user needs to have in order to access the resources in the custom page. 
+These permissions are defined in the `page.properties` file. If your custom page is written in Groovy and uses the Bonita Engine Java APIs, you do not need to specify permissions.
+
+For each REST resource accessed in the page, specify the authorization needed for each method used.  
+You can find examples of the default resources in [`resources-permissions-mapping.properties`](BonitaBPM_platform_setup.md).
+
+The following example shows the permissions defined for a custom page that enables a user to view but not update organization information:
+
+```
+#The name must start with 'custompage_'
+name=custompage_orgViewer
+displayName=Organization viewer
+description=Organization viewer page. You cannot modify the organization from this page.
+resources=[GET|identity/user, GET|identity/personalcontactdata, GET|identity/professionalcontactdata, GET|identity/role, GET|identity/group, GET|identity/membership, GET|customuserinfo/user, GET|customuserinfo/definition, GET|customuserinfo/value] 
+```
+
```

Comment:
```suggestion
When a user is given access to this page because he is in a profile that contains it or that is mapped to an application that contains the page, then this user it granted upon login the permissions associated to these resources (see [REST API authorizations](rest-api-authorization.md) for more details).  

```